Descent3/Game2DLL.cpp

```diff
@@ -136,16 +136,16 @@ static void DUMMYrend_DrawScaledBitmap(int x1, int y1, int x2, int y2, int bm, f
 }
 
 void GetGameAPI(game_api *api) {
-  api->objs = (int *)Objects;
-  api->rooms = (int *)Rooms;
+  api->objs = (int *)std::data(Objects);
```

Comment:
don't like that this is still c-casting to `int*` - this is no safer than before. can the `game_api::...` fields be converted to just be correctly-typed?

Reply:
also, why `std::data` on some of the references but not others?

Reply:
1) I don't know why it's casted at all, so I'm simply not breaking the code.
2) The ones with `std::data` are array `std::array` types while the other are pointers.